src/components/box.js

```diff
@@ -141,6 +141,8 @@
         minHeight: 0,
         position: ({ options: { position } }) =>
           (!isDev && position) || 'unset',
+        zIndex: ({ options: { zIndex, position } }) =>
+          position === 'fixed' ? zIndex : 0,
```

Comment:
The zIndex value should only be applied if a position (other than `static`) is used. I think this should be:
```javascript
position !== 'static' ? zIndex : 'auto'
```